README.md

```
@@ -34,6 +34,7 @@ Learn more at [this project page](https://www.jenkins.io/projects/gsoc/2024/proj
     - [Build](#build)
     - [Using the tool](#using-the-tool)
     - [Setup](#setup)
+    - [Configuring SSH Key](#Configuring-SSH-Key-for-Commit-Signing)
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)